backend/blueprints/spa_api/spa_api.py

```diff
@@ -751,3 +752,60 @@ def api_admin_get_logs(query_params=None):
 @with_query_params(accepted_query_params=[QueryParam(name='id', type_=str, optional=False)])
 def api_admin_get_replay(query_params=None):
     return redirect(f"https://storage.googleapis.com/{FAILED_BUCKET}/{query_params['id']}.replay")
+
+
+# GROUPS
+
+@bp.route('/groups/add', methods=['POST'])
+@require_user
+def create_group():
+    payload = request.get_json(force=True)
+    if payload is None:
+        return jsonify({"Error": "Malformed request"}), 403
+    name = payload['name'] if 'name' in payload else None
+    if 'parent' in payload:
+        parent = payload['parent']
+        if 'game' in payload:
+            entry = SavedGroup.add_game(parent, payload['game'], name)
+        elif 'games' in payload:
+            entry = [SavedGroup.add_game(parent, game, name) for game in payload['games']]
+        else:
+            entry = SavedGroup.add_subgroup(parent, name)
+    else:
+        entry = SavedGroup.create(name)
+    return jsonify({"uuid": entry})
+
+
+@bp.route('/groups/delete', methods=['POST'])
+@require_user
+def delete_group():
+    payload = request.get_json(force=True)
+    if payload is None:
+        return jsonify({"Error": "Malformed request"}), 403
```

Comment:
Why not just raise a calculated error that way we keep track of it inside our metrics?